docs/support/faq.md

```diff
@@ -124,7 +124,7 @@ Yes! Check out this [sample project.](https://github.com/dOrgJelli/lit-protocol-
 
 ### 1. Can more than one condition be added for access control?
 
-Yes! See [boolean logic](../../sdk/access-control/condition-types/boolean-logic) for examples.
```

Comment:
good catches here in FAQ, these are all broken in this section

Reply:
:)